app/client/src/components/shared/Accordion.tsx

```diff
@@ -269,30 +275,32 @@ function AccordionItem({
 
   return (
     <div
+      aria-label={ariaLabel}
       css={accordionItemContainerStyles}
       className={`hmw-accordion`}
       onMouseEnter={(_ev) => addHighlight()}
       onMouseLeave={(_ev) => removeHighlight()}
       onFocus={(_ev) => addHighlight()}
       onBlur={(_ev) => removeHighlight()}
+      onClick={(_ev) => {
+        const newIsOpen = !isOpen;
+        setIsOpen(newIsOpen);
+        onChange(newIsOpen);
+      }}
+      onKeyUp={(ev) => {
+        if (ev.key === 'Enter') {
+          const newIsOpen = !isOpen;
+          setIsOpen(newIsOpen);
+          onChange(newIsOpen);
+        }
+      }}
+      role="listitem"
+      tabIndex={0}
```

Comment:
```suggestion
      role="listitem"
```
Moving these events up to the div level causes an issue, where clicking on content in the accordion item causes the accordion item to be closed. 

![accordion-item-bug](https://user-images.githubusercontent.com/46329268/228966034-a7a92dd2-6225-4d5a-837a-ed2c474cfae0.gif)

Reply:
Good catch, I'll revert that. The only reason I moved that up was for the sake of the screen reader, so the label and tab-able item would be in the same place: currently tabbing over the items doesn't read back the location name. I think, then, the `<header>` tag should have the label, and probably a `role="button"`